test/io/db_config.sql

```diff
@@ -49,9 +53,6 @@ ALTER ROLE other_authenticator SET pgrst.openapi_server_proxy_uri = 'https://oth
 ALTER ROLE other_authenticator SET pgrst.raw_media_types = 'application/vnd.pgrst.other-db-config';
 ALTER ROLE other_authenticator SET pgrst.jwt_secret = 'ODERREALLYREALLYREALLYREALLYVERYSAFE';
 ALTER ROLE other_authenticator SET pgrst.jwt_secret_is_base64 = 'true';
-ALTER ROLE other_authenticator SET pgrst.jwt_role_claim_key = '."other"."role"';
-ALTER ROLE other_authenticator SET pgrst.db_anon_role = 'other';
-ALTER ROLE other_authenticator SET pgrst.db_tx_end = 'rollback-allow-override';
```

Comment:
Ah, you removed these because they wouldn't be set by the function otherwise, right? Perhaps it needs another authenticator user for testing the overrides with the pre config function?

Reply:
Yes, correct. Hm, I thought of a new role but `other_authenticator` already has some redundant testing with `db_config_authenticator`. Applying it to an existing role also ensures the logic is correct.